### PR TITLE
Trying to re-merge pr https://github.com/openxla/xla/pull/6126

### PR DESCRIPTION
**Pull request by Tixxx**
added is_collective_called_computation as a proto field to support serialization from proto.

### CODE REVIEW

File: xla/service/hlo_computation_deduplicator.cc [hunk 1]
```diff
@@ -83,10 +83,12 @@ StatusOr<bool> HloComputationDeduplicator::Run(
     // Ignore entry computation since it is called from outside and computations
     // with large number of instructions or large-size constants due to increase
     // in time taken to stringify.
+    // Also ignore computations that are associated with collective instructions.
     if (comp->IsEntryComputation() || comp->instruction_count() > 128 ||
-        ContainsLargeConstants(comp)) {
+        ContainsLargeConstants(comp) || comp->IsCollectiveCalledComputation()) {
```
Comment (akuegel) [inline xla/service/hlo_computation_deduplicator.cc:88]: Can you please also add a test for this change?

Comment (Tixxx) [reply xla/service/hlo_computation_deduplicator.cc:88]: sorry this was supposed to be a test pr to trigger the CI, i will convert it to a draft for now

**Comment by gbaned** [comment]
Hi @Tixxx This PR is in draft, any update on this? Please. Thank you!

**Comment by Tixxx** [comment]
I will close this, it's not needed anymore. Thanks for reminding me.

**Pull request closed without merging**